draft-ietf-webtrans-http3.md

```diff
@@ -218,6 +218,27 @@ session have alternative mechanisms:
   limiting {{!RFC6585}}.  Unlike the previous method, this signal is directly
   propagated to the application.
 
+## Prioritization
+
+WebTransport does not provide any priority signalling mechanism for streams and
+datagrams within a WebTransport session; such mechanisms will be defined by
+application protocols using WebTransport.
+
+{{!RFC9218}} defines the Priority header which MAY be set in a WebTransport
+CONNECT request or response.  This header SHOULD govern the scheduling of all
+data within the WebTransport session, including WebTransport streams and
+datagrams.
+
+For example, when two WebTransport sessions are established on the same HTTP/3
+connection, each with `Priority: u=1, i`, the scheduler alternates sending data
+from each session.  If an HTTP request is also sent on this connection with
+`Priority: u=0`, the scheduler would send the response data ahead of any data
```

Comment:
This example seems to prescribe an algorithm that can lead to starvation, was that intentional?

Reply:
This is a concrete example of the scheduling guidance in RFC 9218, at least as far as what we say a server should do. We can probably reference that section, which also describes the starvation issues.